review/review.go

```diff
@@ -257,11 +259,13 @@ func buildCommentThreads(commentsByHash map[string]comment.Comment) []CommentThr
 	return threads
 }
 
-// loadComments reads in the log-structured sequence of comments for a review,
+// getCommentsFromNotes parsess in the log-structured sequence of comments for a commit,
```

Comment:
*parses

Also I think you should drop the "in" after parses.

Reply:
Thanks, good catch!

Done.